daprdocs/content/en/operations/monitoring/logging/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+type: docs
+title: "Logging"
+linkTitle: "Logging"
+weight: 400
```

Comment:
This page should have weight so "Metrics" is higher

Reply:
Done